browser/src/robocorp/browser/__init__.py

```diff
@@ -36,20 +36,61 @@ def configure(**kwargs) -> None:
             Run interactions in slow motion.
 
         screenshot:
-            default="only-on-failure",
-            choices=["on", "off", "only-on-failure"],
-            help="Whether to automatically capture a screenshot after each task.",
+            default="only-on-failure"
+            choices=["on", "off", "only-on-failure"]
+            help="Whether to automatically capture a screenshot after each task."
+
+        viewport_size:
+            Size to be set for the viewport. Specified as tuple(width, height).
+
+    Note:
+        See also: `robocorp.browser.browser_context_kwargs` to change other
+        arguments related to the browser context creation.
     """  # noqa
     from ._browser_context import _browser_config
 
     config = _browser_config()
 
     for key, value in kwargs.items():
+        if key == "viewport_size":
+            width, height = value
+            browser_context_kwargs()["viewport"] = {"width": width, "height": height}
+            continue
+
         if not hasattr(config, key):
             raise ValueError(f"Invalid configuration: {key}.")
         setattr(config, key, value)
 
 
+def browser_context_kwargs() -> dict:
```

Comment:
While this _does_ technically describe what is happening, I feel like we could somehow organize this in some way which is more user friendly. Can we maybe combine creating the context explicitly with this method? Or maybe we can use kwargs in the config function as well?

So instead of this, maybe something like....
```python
def configure_context(**kwargs):
    _browser_context.kwargs = kwargs
```

So then the configuration function matches the actual callsite, instead of being just a dict? A minor difference, but something about this makes me not quite satisfied.

Reply:
I agree that `configure_context` receiving the arguments is better (although I made it `browser_context_kwargs.update(kwargs)` instead of always resetting it).